enterprise/migrations/0141_make_enterprisecatalogquery_title_unique.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.24 on 2021-08-27 17:56
+
+from django.db import migrations, models
+from random import randint
+
+def force_unique_title(apps, schema_editor):
+    # I've verified there are no duplicates in Stage and Prod.
+    # This was brought up as a thing devs might need for their local devstack.
+    EnterpriseCatalogQuery = apps.get_model('enterprise', 'EnterpriseCatalogQuery')
+    duplicate_queries = EnterpriseCatalogQuery.objects.raw(
+        'SELECT cq1.title, cq1.id from enterprise_enterprisecatalogquery cq1 '
+        'JOIN (select count(*), title from enterprise_enterprisecatalogquery GROUP BY title having count(*) > 1) cq2 '
+        'ON cq1.title=cq2.title'
+    )
+    for query in duplicate_queries:
+        query.title = f'{query.title}{query.id}{randint(0, 600)}'
```

Comment:
what is the randint guaranteeing here?

Reply:
In thinking about it, idk. I hit a weird case, but that was due to my own manipulation of the db so I don't think this even possible in the wild.

If we applied and then unapplied the migration (which does nothing to the titles, I can't roll those back), it wouldn't matter since the titles would no longer be the same so they wouldn't be duplicates.